backend/src/utils/trust-proxy.ts

```
@@ -0,0 +1,30 @@
+  }
+  if (normalized === 'false') {
+    return false;
+  }
+
+  const hopCount = Number(normalized);
+  if (Number.isFinite(hopCount)) {
+    if (Number.isInteger(hopCount) && hopCount >= 0) {
+      return hopCount;
+    }
+
+    throw new Error('TRUST_PROXY must be a non-negative integer, boolean, or Express trust proxy string');
```

ESLint

[error] Replace `'TRUST_PROXY·must·be·a·non-negative·integer,·boolean,·or·Express·trust·proxy·string'` with `⏎······'TRUST_PROXY·must·be·a·non-negative·integer,·boolean,·or·Express·trust·proxy·string'⏎····`

(prettier/prettier)